### PR TITLE
[slides] chapter linear

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-692/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

File: chapter_linear-networks/softmax-regression-scratch.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ b = tf.Variable(tf.zeros(num_outputs))
 
 ## 定义softmax操作
 
-在实现softmax回归模型之前，让我们简要地回顾一下`sum`运算符如何沿着张量中的特定维度工作，如 :numref:`subseq_lin-alg-reduction` 和 :numref:`subseq_lin-alg-non-reduction` 所述。给定一个矩阵`X`，我们可以对所有元素求和（默认情况下），也可以只求同一个轴上的元素，即同一列（轴0）或同一行（轴1）。如果 `X` 是一个形状为 `(2, 3)` 的张量，我们对列进行求和，则结果将是一个具有形状 `(3,)` 的向量。当调用sum运算符时，我们可以指定保持在原始张量的轴数，而不折叠求和的维度。这将产生一个具有形状 `(1, 3)` 的二维张量。
+在实现softmax回归模型之前，让我们简要地回顾一下`sum`运算符如何沿着张量中的特定维度工作，如 :numref:`subseq_lin-alg-reduction` 和 :numref:`subseq_lin-alg-non-reduction` 所述。
+[**给定一个矩阵`X`，我们可以对所有元素求和**]（默认情况下），也可以只求同一个轴上的元素，即同一列（轴0）或同一行（轴1）。如果 `X` 是一个形状为 `(2, 3)` 的张量，我们对列进行求和，则结果将是一个具有形状 `(3,)` 的向量。当调用sum运算符时，我们可以指定保持在原始张量的轴数，而不折叠求和的维度。这将产生一个具有形状 `(1, 3)` 的二维张量。
```
Comment (astonzhang) [inline chapter_linear-networks/softmax-regression-scratch.md:71]: 换行后git diff不容易只把change显示出来。为了review方便，最好避免不必要的换行，这样一眼就能看出来这里的改动只有增加了`[** **]`

**Comment by mli** [comment]
Job d2l-zh/PR-692/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/9 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

**Comment by mli** [comment]
Job d2l-zh/PR-692/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-692/

File: chapter_linear-networks/softmax-regression-concise.md [hunk 2]
```diff
@@ -81,7 +83,7 @@ $$
 $$
 
 我们也希望保留传统的softmax函数，以备我们需要评估通过模型输出的概率。
-但是，我们没有将softmax概率传递到损失函数中，而是在交叉熵损失函数中传递未归一化的预测并同时计算softmax及其对数，这是一件聪明的事情 ["LogSumExp技巧"](https://en.wikipedia.org/wiki/LogSumExp)。
+但是，我们没有将softmax概率传递到损失函数中，而是[**在交叉熵损失函数中，传递未归一化的预测，并同时计算softmax及其对数**]，这是一件聪明的事情 ["LogSumExp技巧"](https://en.wikipedia.org/wiki/LogSumExp)。
```
Comment (astonzhang) [inline chapter_linear-networks/softmax-regression-concise.md:86]: 在交叉熵损失函数中，传递未归一化的预测
-> 在交叉熵损失函数中传递未归一化的预测

**Pull request merged**